blocks/v2-all-trucks/v2-all-trucks.js

```diff
@@ -0,0 +1,183 @@
+import { createElement } from '../../scripts/common.js';
+
+// Define break point/s
+const MQMobile = window.matchMedia('(max-width: 744px)');
+const blockName = 'v2-all-trucks';
+
+export default function decorate(block) {
+  const pageDescriptionHeading = document.querySelector('.v2-all-trucks-container > div > h1');
```

Comment:
you can replace `'.v2-all-trucks-container > div > h1'` to `${blockName}-container > div > h1`

Also same at line 15, 20, 26